src/components/Observations/index.tsx

```diff
@@ -31,7 +34,17 @@ export default function PlantsDashboard(): JSX.Element {
       onPlantingSites={onPlantingSites}
       isEmptyState={!plantingSites?.length || !hasObservations}
     >
-      <div>placeholder for selected planting site {selectedPlantingSite?.id}</div>
+      {hasObservations ? (
+        <div>placeholder for selected planting site {selectedPlantingSite?.id}</div>
+      ) : (
+        <Card style={{ margin: 'auto' }}>
+          <EmptyStateContent
+            title={strings.OBSERVATIONS_EMPTY_STATE_TITLE}
+            subtitle={[strings.OBSERVATIONS_EMPTY_STATE_MESSAGE_1, strings.OBSERVATIONS_EMPTY_STATE_MESSAGE_2]}
+            styles={EMPTY_STATE_CONTENT_STYLES}
```

Comment:
Curious whether this is something the EmptyStateContent component should apply by default if `styles` is not supplied?

Reply:
Probably should. This is some very old code, unrelated to this PR.